tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AWS/Lambda/LambdaCommon.cs

```diff
@@ -25,10 +25,15 @@ internal abstract class LambdaCommon
 
     internal static Scope CreatePlaceholderScope(Tracer tracer, NameValueHeadersCollection headers)
     {
-        var spanContext = SpanContextPropagator.Instance.Extract(headers);
-        TelemetryFactory.Metrics.RecordCountSpanCreated(MetricTags.IntegrationName.AwsLambda);
-
-        var span = spanContext != null ? tracer.StartSpan(PlaceholderOperationName, tags: null, parent: spanContext, serviceName: PlaceholderServiceName, addToTraceContext: false) : tracer.StartSpan(PlaceholderOperationName, tags: null, serviceName: PlaceholderServiceName, addToTraceContext: false);
+        var context = SpanContextPropagator.Instance.Extract(headers);
+        Baggage.Current.Merge(context.Baggage);
+
+        var span = tracer.StartSpan(
```

Comment:
I noticed that you removed the ternary operation. Is the result the same regardless in both cases where the SpanContext is null and when SpanContext is not null?

Reply:
Yes, `parent` is an optional parameter with default value `null`, so this code was calling the same method either way, but skipping `parent` if it was `null`.

@bouwkast suggested this in https://github.com/DataDog/dd-trace-dotnet/pull/6041#discussion_r1811145691, but it was missed before merging.